src/Spice86.Core/Emulator/InterruptHandlers/Input/Keyboard/KeyboardInt16Handler.cs

```diff
@@ -60,80 +55,92 @@ private void Unsupported(int operation) {
     public override byte VectorNumber => 0x16;
 
     /// <summary>
-    ///     Emits a minimal INT 16h handler stub into guest RAM that handles AH=00h (GetKeystroke) in-place
-    ///     and dispatches all other AH values back to the C# handler via a callback.
+    /// Emits a minimal INT 16h handler stub into guest RAM.
     /// </summary>
     /// <remarks><![CDATA[
-    ///     Assembled control flow (labels for readability):
-    ///     L_HANDLER:
+    /// Handles AH=00h (Get Keystroke) locally with a busy-wait loop driven by a callback that sets ZF,
+    /// and dispatches all other AH values back to the C# handler.
+    /// 
+    /// Pseudo-assembly (labels for readability):
+    /// 
+    ///   L_HANDLER:
     ///     cmp ah, 00h
     ///     jz  L_AH00
     /// 
-    ///     L_DEFAULT:
-    ///     int 0A4h   ; DefaultDispatch -> calls C# Run()
+    ///   L_DEFAULT:
+    ///     int cbDefaultDispatch   ; calls C# Run() to handle AH != 00h
```

Comment:
this is wrong, its not an interrupt it is a callback. Same for the rest of the comment referencing callbacks as INT